test/Microsoft.Process.Environment.Tests/EnvironmentManagerCentennialTests.cpp

```diff
@@ -0,0 +1,26 @@
+﻿#include <pch.h>
+#include <EnvironmentManagerCentennialTests.h>
+
+using namespace winrt::Microsoft::ProjectReunion;
+
+namespace ProjectReunionCppTest
+{
+    void EnvironmentManagerCentennialTests::CentennialTestGetForProcess()
+    {
+        EnvironmentManager environmentManager = EnvironmentManager::GetForProcess();
+        VERIFY_IS_NOT_NULL(environmentManager);
```

Comment:
Can you do test more than just you got anything? Verify you got the right type of environment manager?

Reply:
That is a good idea.  That would require me adding the Scope enum to the IDL so I can expose it to the testing code.

Is it okay to alter the API space after the initial review from my first PR?

Reply:
I'd think so in this case. But check with your API rep. Maybe just needs a spec update and a simple (offline/email) API Review of the change. OR updated spec and your API rep brings the change to API Review and lets you know if there's any problem.

Reply:
Instead, write a test that does something with the process-scoped env manager that shouldn't be visible to another process.  Like:

1. Get the env manager here
2. Set a property on it
3. Read the real registry and verify that the set did not go through to the persistent hive
4. Create a child process from here
5. Verify that the set property from 2 does show up in the new child process

Reply:
Also feel free to _not_ have the "verify this is not null." If it ever does become null, the follow-on "set" calls will access-violate and tell you the GetForProcess didn't return something.

Reply:
> Instead, write a test that does something with the process-scoped env manager that shouldn't be visible to another process. Like:
> 
> 1. Get the env manager here
> 2. Set a property on it
> 3. Read the real registry and verify that the set did not go through to the persistent hive
> 4. Create a child process from here
> 5. Verify that the set property from 2 does show up in the new child process

I'm confused.  I haven't implemented SetEnvironmentVariable yet, so I can't call it.  Additionally, there are no properties to set.  If I did set a property, why would it go to the registry?

Reply:
> Also feel free to _not_ have the "verify this is not null." If it ever does become null, the follow-on "set" calls will access-violate and tell you the GetForProcess didn't return something.

Isn't that the point of the GetFor[Process\User\Machine] calls, to get a non-null object?

Reply:
ping @jonwis

Reply:
I see. This PR only sets up the scaffolding for the code, and the methods that "do the things" are still coming.

I guess for now, I'd add modify the test to actually detect the "not implemented" state, like:

```c++
VERIFY_THROWS(manager.SetEnvironmentVariable(), winrt::hresult_not_implemented);
```